pkg/distro/fedora/images.go

```diff
@@ -375,12 +378,14 @@ func imageInstallerImage(workload workload.Workload,
 
 	d := t.arch.distro
 
-	img.ISOLabelTempl = d.isolabelTmpl
 	img.Product = d.product
 	img.OSName = "fedora"
 	img.OSVersion = d.osVersion
 	img.Release = fmt.Sprintf("%s %s", d.product, d.osVersion)
 
+	// We don't know the variant of the OS pipeline being installed
+	img.ISOLabel = fmt.Sprintf(ISO_LABEL, img.Product, img.OSVersion, "Unknown", img.Platform.GetArch())
```

Comment:
I'm wondering if it would not make sense to instead set the `img.Variant = "Unknown"` and then construct `ISOLabel` in the same way as in other image types.

Reply:
Agreed, better than the above.

Reply:
Added a commit that resolves this.